docs.md

```diff
@@ -0,0 +1,16 @@
+# XAQT Library
+*TODO (cw|4.28.2018) create godocs instead of this file...these are mostly notes for myself during development.*
+
+# Public Types
+* **`xaqt.Context`**: entrypoint into all functionality. TODO Propose renaming?
```

Comment:
Yeah despite naming it that, I don't love it... especially now that we're using the `Context` library.

Reply:
yeah, i feel you, naming is hard and often doesn't feel like it holds up over time as the code progresses-- luckily this is a painless refactor at this point, so if we decide on a good rename we can change it. i won't do it in this PR since its not super relevant...but we can think it over :)